pds_doi_service/core/outputs/osti.py

```diff
@@ -23,7 +23,7 @@ def create_osti_doi_draft_record(self, doi: Doi):
         doi_fields['publication_date'] = doi.publication_date.strftime('%Y-%m-%d')
         doi_fields['keywords'] = "; ".join(doi.keywords)
         renderer = pystache.Renderer()
-        template_path = join(self._root_dir, 'config', 'DOI_template_20200407-mustache.xml')
+        template_path = join(dirname(__file__), 'DOI_template_20200407-mustache.xml')
```

Comment:
Since its no longer being used, you can remove the `__init__` method which declares `self._root_dir`